.github/workflows/centraldb_docker_publish.yaml

```diff
@@ -34,8 +42,17 @@ jobs:
         export IMG=kubeflownotebookswg/centraldashboard
         make docker-build
 
+    - name: Run CentralDashboard push on Version change
+      id: version
+      if: steps.filter.outputs.version == 'true'
+      run: |
+        export TAG=$(cat releasing/version/VERSION)
+        cd components/centraldashboard
+        export IMG=kubeflownotebookswg/centraldashboard
+        make docker-push
+
     - name: Run CentralDashboard push
-      if: github.event_name == 'push'
+      if: github.event_name == 'push' && steps.version.conclusion == 'skipped'
```

Comment:
Does this mean that this step will NOT be run if the `releasing/version/VERSION` file is not changed?

Let's still run that step, even if we pushed the image with a specific tag above. We still want to push the images with the tag being the git commit that got merged

Reply:
The above means that this step only runs if the `version` push step is skipped. So yes if the `releasing/version/VERSION` file is changed this step will not run.

Reply:
I added a commit and removed the `&& steps.version.conclusion == 'skipped'` part from all the workflows.